snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: plexmediaserver
-version: 1.2.2.2857-d34b464-1
+version: 1.2.2.2857-d34b464
```

Comment:
Please don't drop -1. That is actually the snap revision number which gets bumped whenever upstream does not change but the packaging.